src/Core/Config.cs

```diff
@@ -46,6 +47,13 @@ public Config WithPathsRelativeTo(string baseDirectory)
 
         public bool HasCustomTestRunnerCommand => !string.IsNullOrEmpty(CustomTestRunnerCommand);
 
+	    public string GetSolutionFolder()

```

Comment:
I like how you extracted this into a method here 👍

Reply:
Thanks!

Reply:
No problem, that's why we have code reviews :)

No more action required from you, I'll merge the PR and make a new Fettle release soon (hopefully tomorrow).

Thanks for contributing 👍

Reply:
Ok thanks. This was my first contribution to a GitHub project, and I liked it. Perhaps I can contribute more in the future!